frontend/src/modules/Home/Components/Home.tsx

```diff
@@ -1,59 +1,97 @@
-import {
-  StyledBackground,
-  StyledLeftPanel,
-  StyledRightPanel,
-} from 'Home/Styles/Home.style'
-import { adminSignIn } from '@fire'
+import { StyledBackground } from 'Home/Styles/Home.style'
 import { Box, Button, Typography } from '@mui/material'
 import matchimg from '@assets/img/matching.svg'
 import { Link } from 'react-router-dom'
 import { SURVEY_PATH } from '@core/Constants'
 
+import { ReactComponent as CornellSeal } from '@assets/img/CornellSealWhite.svg'
+
 export const Home = () => {
   return (
     <StyledBackground>
-      <StyledLeftPanel>
+      <Box
+        sx={{
+          display: 'flex',
+          flexFlow: 'column nowrap',
+          justifyContent: 'center',
+          height: '100%',
+          width: {
+            xs: '100%',
+            md: '70%',
+          },
+          alignItems: 'left',
+          padding: '0 0 0 4rem',
+        }}
+      >
+        <Box
+          sx={{
+            display: 'flex',
+            flexFlow: 'row nowrap',
+            gap: '1rem',
+            justifyContent: 'center',
+            alignItems: 'center',
+          }}
+        >
+          <Box
+            sx={{
+              width: '100%',
+              maxWidth: '50px',
+            }}
+          >
+            <CornellSeal />
+          </Box>
```

Comment:
Just changed to 73px. I saw this too and originally tried using that but I wasn't able to crop the image. As, I could convert to JPG and then crop but its just a white block since JPG doesn't remove background. Might be possible in a photoshop tool so perhaps Prachi/designer could work on that and replace it if need be.